jmx-exporter-bitnami.yaml

```diff
@@ -0,0 +1,114 @@
+package:
+  name: jmx-exporter-bitnami
+  version: "1.1.0"
+  epoch: 0
+  description: "Bitnami for Prometheus JMX Exporter"
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates
+      - curl
+      - maven
+      - openjdk-17-default-jdk
+      - procps
+      - zlib
+  environment:
+    LANG: en_US.UTF-8
+    JAVA_HOME: /usr/lib/jvm/java-17-openjdk
+
+var-transforms:
+  # Extract the major version from the package version.
+  - from: ${{package.version}}
+    match: ^(\d+).*
+    replace: $1
+    to: major-version
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/prometheus/jmx_exporter
+      tag: ${{package.version}}
+      expected-commit: 00fd35dc2e0a03eefe49ef1182280fc9319bbe70
+
+  - name: build-intall-jars
+    # Build the project using Maven
+    runs: |
+      ./mvnw clean package -DskipTests
+
+  - name: create-required-dirs
+    # Create the required directories for a Bitnami-like layout
+    runs: |
+      mkdir -p ${{targets.contextdir}}/opt/bitnami
+      mkdir -p ${{targets.contextdir}}/opt/bitnami/jmx-exporter
+      mkdir -p ${{targets.contextdir}}/opt/bitnami/java
+      mkdir -p ${{targets.contextdir}}/opt/bitnami/jmx-exporter/licenses
+      mkdir -p ${{targets.contextdir}}/tmp/bitnami/pkg/cache/
+
+  - name: copy-files-to-dest
+    # Bitnami doesn't like symlinks
+    runs: |
+      cp -R ./examples ${{targets.destdir}}/opt/bitnami/jmx-exporter/
+      cp ./LICENSE ${{targets.destdir}}/opt/bitnami/jmx-exporter/
+
+      # Copy the Java 17 installation into /opt/bitnami/java
+      for item in bin conf include jre legal lib release; do
+        cp -R /usr/lib/jvm/java-17-openjdk/$item ${{targets.destdir}}/opt/bitnami/java/
+      done
+
+      # Copy built JARs for agent and standalone server
+      cp ./jmx_prometheus_javaagent/target/jmx_prometheus_javaagent-${{package.version}}.jar \
+          ${{targets.destdir}}/opt/bitnami/jmx-exporter/jmx_prometheus_javaagent.jar
+      cp ./jmx_prometheus_standalone/target/jmx_prometheus_standalone-${{package.version}}.jar \
+          ${{targets.destdir}}/opt/bitnami/jmx-exporter/jmx_prometheus_standalone.jar
+
+  - uses: bitnami/compat
+    with:
+      image: jmx-exporter
+      version-path: ${{vars.major-version}}/debian-12
+
+test:
+  environment:
+    contents:
+      packages:
+        - curl
+        - openjdk-17-default-jdk
```

Comment:
I believe the JVM should be enough. Let's also create a variable for the `java-version` and re-use